x/params/subspace/subspace.go

```diff
@@ -64,11 +64,24 @@ func (s Subspace) kvStore(ctx sdk.Context) sdk.KVStore {
 	return ctx.KVStore(s.key).Prefix(append(s.name, '/'))
 }
 
-// Returns a KVStore identical with ctx.TransientStore(s.tkey).Prefix()
-func (s Subspace) transientStore(ctx sdk.Context) sdk.KVStore {
+// Returns a transient store for modification
+func (s Subspace) modifiedStore(ctx sdk.Context) sdk.KVStore {
```

Comment:
I guess since this method is private, it doesn't matter, but I liked transient as part of the name.